showcase/app/controllers/components/copy/button.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import { scheduleOnce } from '@ember/runloop';
+
+import type Owner from '@ember/owner';
+import type { Registry as Services } from '@ember/service';
+
+import type { ComponentsCopyButtonModelSignature } from 'showcase/routes/components/copy/button';
+
+export default class CopyButtonController extends Controller {
+  declare model: ComponentsCopyButtonModelSignature;
```

Comment:
Note: this is what tells the typescript compiler that the model exists, otherwise it is unknown.